### ISSUE TITLE
EDSC-968: Multiple bounding rectangles on map, but only one in collection details text

### ISSUE THREAD

**Issue by earthdata-tasktop** [nasa/edsc-search-test-config-3#606]
[https://search.earthdata.nasa.gov/search/collection-details?p=C1214560151-JAXA&amp;m=2.293010499999994!-87.18745000000002!1!1!0!&amp;q=jaxa](https://search.earthdata.nasa.gov/search/collection-details?p=C1214560151-JAXA&m=2.293010499999994!-87.18745000000002!1!1!0!&q=jaxa)

For the collection GRFM_NASDA_NASA in OPS, go to the Collection Details page.  
A single bounding rectangle is listed in the collection details text (Bounding Rectangle: (27°, 92°, -20°, 151°), but three bounding rectangles are shown on the map.

See attached screen shot.

**Comment by earthdata-tasktop** [comment]
(Comment from Bo Sun):

##### Steps to Verify

In UAT: [https://search.uat.earthdata.nasa.gov/search/collection-details?cmr_env=prod&amp;amp;p=C1214560151-JAXA&amp;amp;q=C1214560151-JAXA](https://search.uat.earthdata.nasa.gov/search/collection-details?cmr_env=prod&p=C1214560151-JAXA&q=C1214560151-JAXA)

you should see all three Bounding Rectangle are displayed on the collection detail page.

**Comment by earthdata-tasktop** [comment]
(Comment from Kathleen Carr):

See third screen shot for a case where there are four bounding rectangles - all are shown on the map and all are shown in the text

**Comment by earthdata-tasktop** [comment]
(Comment from Kathleen Carr):

I searched for Global Rain Forest Mapping Project (GRFM) collections in UAT.   I clicked on the Collection Details for several of these collections.   The number of bounding rectangles shown on the map corresponded to the number of bounding rectangles shown in the Spatial Coordinates text.   See attached screen shots for examples with three bounding rectangles and four bounding rectangles.
